cmd/lp-cc/cc.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/ScottSallinen/lollipop/enforce"
+	"github.com/ScottSallinen/lollipop/graph"
+)
+
+const EMPTYVAL = math.MaxUint32
+
+type VertexProperty struct {
+	Value   uint32
+	Scratch uint32 // Intermediary accumulator
+}
+
+func (p *VertexProperty) String() string {
+	return fmt.Sprintf("%d", p.Value)
+}
+
+type EdgeProperty struct {}
+
+type MessageValue uint32
+
+func MessageAggregator(dst *graph.Vertex[VertexProperty, EdgeProperty], didx, sidx uint32, data MessageValue) (newInfo bool) {
+    input := uint32(data)
+    if input > dst.Id { // This means ininitalization
+        input = dst.Id
+    }
```

Comment:
```suggestion
  input := uint32(data)
  if input > dst.Id { // This means ininitalization
      input = dst.Id
  }
```

Check your indentation here. If you are using VSCode, I also suggest installing and setting up the Go extension from the extension manager -- this will help automatically format code and make sure your changes compile. It is very handy!